hazelcast-spring/src/main/resources/hazelcast-spring-5.1.xsd

```diff
@@ -4637,6 +4638,92 @@
         </xs:attribute>
     </xs:complexType>
 
+    <xs:complexType name="in-memory-tier">
+        <xs:annotation>
+            <xs:documentation>
+                Defines configuration for the in-memory tier of a tiered store.
+            </xs:documentation>
+        </xs:annotation>
+        <xs:all>
+            <xs:element name="capacity" type="xs:unsignedLong" minOccurs="0" default="256">
+                <xs:annotation>
+                    <xs:documentation>
+                        Capacity in MB, i.e., how much main memory should this tier consume at most.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+        </xs:all>
+    </xs:complexType>
+
+    <xs:complexType name="disk-tier">
+        <xs:annotation>
+            <xs:documentation>
+                Defines configuration for the disk tier of a tiered store.
+            </xs:documentation>
+        </xs:annotation>
+        <xs:all>
+            <xs:element name="base-dir" type="xs:string" minOccurs="0" default="tstore">
+                <xs:annotation>
+                    <xs:documentation>
+                        Base directory for storing files.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+            <xs:element name="block-size" type="xs:unsignedInt" minOccurs="0" default="4096">
+                <xs:annotation>
+                    <xs:documentation>
+                        Disk block/sector size in bytes. The default is 4096. The minimum is 64.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+        </xs:all>
+        <xs:attribute name="enabled" type="xs:boolean" use="required">
+            <xs:annotation>
+                <xs:documentation>
+                    True to enable using disk as the second memory tier, false otherwise.
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="capacity" type="xs:unsignedLong" use="required">
+            <xs:annotation>
+                <xs:documentation>
+                    Disk capacity in GB, i.e., how much disk space should be consumed at most.
```

Comment:
Currently we don't. The reasoning was that since disk capacities can vary a lot, putting some default might be too small or too large in some cases. Should we put a default here? Maybe 4 GB disk capacity would be the default?

Reply:
Maybe by default set Long.MAX_VALUE?

Reply:
then the default would not be usable at all, no?